app/src/main/java/com/example/internetshop/data/retrofitapi/CategoryApi.kt

```diff
@@ -11,6 +11,6 @@ interface CategoryApi {
 
     @GET("/products/category/{categoryName}")
     fun getCategoryProducts(
-        @Path("categoryName") categoryName: String
+        @Path("categoryName") categoryRequest: String
```

Comment:
rename back to `categoryName` or to `name`

Reply:
fixed